doc/api/esm.md

```diff
@@ -235,30 +238,34 @@ absolute URL strings.
 import fs from 'node:fs/promises';
 ```
 
-## Import assertions
+## Import attributes
 
 <!-- YAML
 added:
   - v17.1.0
   - v16.14.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/50140
+    description: Switch from Import Assertions to Import Attributes.
 -->
 
 > Stability: 1 - Experimental
```

Comment:
There’s no such thing as a JS spec, and the proposal never made it to stage 4 anyway. 
```suggestion
> Stability: 1.1 - Active development

> This feature was previously named "Import assertions", and using the `assert`
> keyword instead of `with`. Any uses in code of the prior `assert` keyword should
> be updated to use `with` instead.
```

Reply:
Sure, your version looks good.